### DISCUSSION TITLE
Windows on ARM support?

### DISCUSSION THREAD

**Discussion by Benjin** [ramensoftware/windhawk#210] [Ideas]
Snapdragon-based laptops are starting to roll out, and I'd love to be able to use this on mine.  Currently, setup terminates with a failure to start the service:

![image](https://github.com/ramensoftware/windhawk/assets/1609827/ca996b93-2106-46a7-9043-0908719daf5a)

**Comment by m417z** [comment]
It's a known limitation, you can see and follow the relevant issue here: https://github.com/ramensoftware/windhawk/issues/11.

**Comment by Benjin** [comment]
Following.  But also, last update over a year ago.

—
Benjin Dubishar
________________________________

**Reply by m417z** [reply]
Well, no work was done in this area, and the status that's described in the comments didn't change.

**Comment by qbit-soul** [comment]
@Benjin @m417z was experimenting with agents to automate my workflow, I somehow noticed out of blue my agent posted on this particular discussion. Sorry for any type of inconvenience.
